### ISSUE TITLE
About Split.

### ISSUE THREAD

**Issue by Chancerst** [Barbosik/MultiOgar#27]
Split is too slow than vanilla agar.io.
Please fix it.
This is the Problem video.

[Desktop 2016.06.11 - 10.01.27.05.zip](https://github.com/Barbosik/MultiOgar/files/310033/Desktop.2016.06.11.-.10.01.27.05.zip)

**Comment by Barbosik** [comment]
I know it. The speed for split and eject should be tuned. I'm working on it

**Comment by Chancerst** [comment]
@Barbosik Thank you!

**Comment by Barbosik** [comment]
I increased speed. Not sure if it's the same as in vanilla, it's hard to compare :)
Could you please check it?

**Comment by Chancerst** [comment]
@Barbosik OK. I'll Check :)

**Comment by Chancerst** [comment]
@Barbosik Still, it slow..
Please Compare Agar.io Split Speed and Multiogar Split Speed. (ㅇ.ㅇ)

**Comment by MertOtrk** [comment]
Remove "split animation".

**Comment by Chancerst** [comment]
@Barbosik 
[bandicam 0075.zip](https://github.com/Barbosik/MultiOgar/files/310076/bandicam.0075.zip)
It is Vanilla agar.io Split Speed.

**Comment by Barbosik** [comment]
Changed again, looks like close to vanilla. Check it please

**Comment by Chancerst** [comment]
@Barbosik OK, I'll check

**Comment by Chancerst** [comment]
@Barbosik At the end motion, it looks slowly.
At the start speed and At the end speed should same speed.

**Comment by Chancerst** [comment]
@Barbosik Split (2, 4, 16) 
[Desktop 2016.06.11 - 13.42.40.06.zip](https://github.com/Barbosik/MultiOgar/files/310097/Desktop.2016.06.11.-.13.42.40.06.zip)
Reach the Virus
[Desktop 2016.06.11 - 13.43.34.07.zip](https://github.com/Barbosik/MultiOgar/files/310098/Desktop.2016.06.11.-.13.43.34.07.zip)

**Comment by Chancerst** [comment]
@Barbosik Split (2, 4, 16)
[Desktop 2016.06.11 - 13.46.40.08.zip](https://github.com/Barbosik/MultiOgar/files/310100/Desktop.2016.06.11.-.13.46.40.08.zip)
Reach the Virus
[Desktop 2016.06.11 - 13.47.25.09.zip](https://github.com/Barbosik/MultiOgar/files/310101/Desktop.2016.06.11.-.13.47.25.09.zip)

(Multiogar)


Can you see Diffrerence?

**Comment by Barbosik** [comment]
What is wrong with split?

About virus, it's because collision is disabled when you throw on the virus, so some cells fly inside another cells, and on the next second collision is enabled again and these cells which were inside the other extruded away. 
Actually, this behavior is used for split, but it's enabled for virus by mistake. I will fix it later

**Comment by Chancerst** [comment]
@Barbosik You can't see Difference about split motion?

**Comment by Barbosik** [comment]
Can you show please comparison between vanilla and multiogar?
I see just multiogar video, but I don't know how it looks for the same conditions (cell mass, split count) in vanilla. At a glance it looks similar. Where is difference exactly?

**Comment by Barbosik** [comment]
**Chancerst**: I don't think that vanilla uses fast speed at the end.
I recorded video from vanilla with split from 100 mass.
Here is my measurements result: http://imgur.com/oYp2Yyk

![Split 100](http://i.imgur.com/oYp2Yyk.png)

X - rest of the boosted distance
Y - boost step

**Comment by Chancerst** [comment]
@Barbosik Yes. That is OgarProject Split motion.
But, I Think OgarProject Split Motion is smoother than Multi-ogar And Similar Vanilla agar.io.
Do you think Multiogar-master has Vanilla Agar.io Split Motion?
I think you should Compare Ogarproject split and Multi-ogar split.
OgarProject split has a little slow at the end,
But Multiogar split has a slow more than Ogar project.
So, i think Multiogar split has little strange.
Can i request once time to change ogarproject split source this Serverpack?

**Comment by Barbosik** [comment]
I decreased collision disable time to 0.5 sec. Test it.

I just want to make physics more close to vanilla, not to OgarProject ;)

**Comment by Barbosik** [comment]
split/eject speed and collision behavior fixed to vanilla physics

**Comment by Chancerst** [comment]
@Barbosik So you mean it fixed?

**Comment by Barbosik** [comment]
yes, now it works exactly the same as on vanilla servers.
I tested it by recording video on vanilla server for cell split with mass 100 and with mass 2256.
I compared it tick by tick with MultiOgar - both works the same.
The same distance, the same speed, the same collision behavior.
I didn't find any difference for mass 100 and mass 2256.

Also I recorded video on vanilla server for virus throw in the same condition as in your video.
I recorded it for two cells with mass 734 and 729.
Then I compared it with MultiOgar. Both disable collision for the same time.
So both works in the same way.
